packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts

```diff
@@ -38,27 +40,114 @@ const GlobalClassNames = {
   custom: 'ms-Panel--custom'
 };
 
-const panelSize = {
-  width: {
-    xs: '272px',
-    sm: '340px',
```

Comment:
Replaced this const with other two for a better semantic meaning and easier writing... instead of `panelSize.width.sm` now we have `panelWidth.sm`